src/Npgsql/Npgsql.csproj

```diff
@@ -19,25 +17,23 @@
     <ProjectReference Include="../Npgsql.SourceGenerators/Npgsql.SourceGenerators.csproj" OutputItemType="Analyzer" ReferenceOutputAssembly="false" />
   </ItemGroup>
 
-  <ItemGroup Condition=" '$(TargetFramework)' == 'net7.0' ">
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net8.0' ">
     <PackageReference Include="Microsoft.CodeAnalysis.PublicApiAnalyzers" PrivateAssets="All" />
   </ItemGroup>
 
   <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
     <PackageReference Include="Microsoft.Bcl.HashCode" />
+    <PackageReference Include="System.Memory" />
```

Comment:
Is this needed, i.e. how come we could build without it before now?

Reply:
No, I did this to support ReadOnlyMemory<byte> and Memory<byte> in ByteaHandler while enabling functionality for .NET Standard 2.1 builds but since this is for .NET Standard 2.0 none of the TFMs we're dropping will benefit from it so we probably shouldn't do this.

Reply:
Ok, We can avoid directly importing it, since System.Text.Json (which we already import) depends on it and we don't have any version requirements.